packages/ai-mcp/tests/pool.test.ts

```
@@ -0,0 +1,55 @@
+import { describe, expect, it } from 'vitest'
+import { makeServerWithWeatherTool } from './helpers/in-memory-server'
+import { createMCPClients } from '../src/pool'
```

ESLint

[error] `../src/pool` import should occur before import of `./helpers/in-memory-server`

(import/order)